tests/UnifiedSpecTests/Context.php

```diff
@@ -463,7 +463,7 @@ private function createSession(string $id, stdClass $o): void
 
     private function createBucket(string $id, stdClass $o): void
     {
-        Util::assertHasOnlyKeys($o, ['id', 'database', 'bucketOptions']);
+        Util::assertHasOnlyKeys($o, ['id', 'database', 'bucketOptions', 'disableMD5']);
```

Comment:
```suggestion
        Util::assertHasOnlyKeys($o, ['id', 'database', 'bucketOptions']);
```

Was this change actually required for anything? This is the code path for `createEntities`. The UTF schema doesn't allow `disableMD5` here, so it would only appear within `bucketOptions` if at all.

That said, it doesn't even look like we have spec tests using `bucketOptions`.